From 94d97ff4879109a4e6f154bd736aaaaaac4c5d3b Mon Sep 17 00:00:00 2001
From: Jacob Chen <jacob2.chen@rock-chips.com>
Date: Mon, 31 Jul 2017 14:19:51 +0800
Subject: [PATCH] arm64: rockchip_linux_defconfig: change to use v4l2 rga

At present no component in linux depend on drm RGA, i'd like to remove it

Change-Id: Ibf531641d3dd30bd87f83dbfe1fd06af2d3dffcb
Signed-off-by: Jacob Chen <jacob2.chen@rock-chips.com>
---
 arch/arm64/configs/rockchip_linux_defconfig | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/arch/arm64/configs/rockchip_linux_defconfig b/arch/arm64/configs/rockchip_linux_defconfig
index e40a1b6c7651..382a52c1fe1c 100644
--- a/arch/arm64/configs/rockchip_linux_defconfig
+++ b/arch/arm64/configs/rockchip_linux_defconfig
@@ -283,6 +283,8 @@ CONFIG_USB_VIDEO_CLASS=y
 CONFIG_V4L_PLATFORM_DRIVERS=y
 CONFIG_SOC_CAMERA=y
 CONFIG_VIDEO_RK_CIF_ISP10=y
+CONFIG_V4L_MEM2MEM_DRIVERS=y
+CONFIG_VIDEO_ROCKCHIP_RGA=y
 # CONFIG_MEDIA_SUBDRV_AUTOSELECT is not set
 # CONFIG_VIDEO_IR_I2C is not set
 # CONFIG_DVB_AU8522_V4L is not set
@@ -293,7 +295,6 @@ CONFIG_DRM=y
 CONFIG_DRM_LOAD_EDID_FIRMWARE=y
 CONFIG_DRM_DMA_SYNC=y
 CONFIG_DRM_ROCKCHIP=y
-CONFIG_ROCKCHIP_DRM_RGA=y
 CONFIG_ROCKCHIP_DW_HDMI=y
 CONFIG_ROCKCHIP_DW_MIPI_DSI=y
 CONFIG_ROCKCHIP_ANALOGIX_DP=y
-- 
2.35.3

